pkg/store/verifiable/store.go

```diff
@@ -229,6 +231,62 @@ func (s *StoreImplementation) GetPresentations() ([]*Record, error) {
 	return s.getAllRecords(presentationNameDataKey(""), getPresentationName)
 }
 
+// RemoveCredential removes the verifiable credential and its records containing given name.
+func (s *StoreImplementation) RemoveCredential(name string) error {
+	if name == "" {
+		return errors.New("credential name is mandatory")
+	}
+
+	id, err := s.GetCredentialIDByName(name)
+	if err != nil && !errors.Is(err, storage.ErrDataNotFound) {
+		return fmt.Errorf("get credential id using name : %w", err)
+	}
+
+	err = s.remove(id, credentialNameDataKey(name))
+	if err != nil {
+		return fmt.Errorf("unable to delete credential : %w", err)
+	}
+
+	return nil
+}
+
+// RemovePresentation removes the verifiable presentation and its records containing given name.
+func (s *StoreImplementation) RemovePresentation(name string) error {
+	if name == "" {
+		return errors.New("presentation name is mandatory")
+	}
+
+	id, err := s.GetPresentationIDByName(name)
+	if err != nil && !errors.Is(err, storage.ErrDataNotFound) {
```

Comment:
same here `!errors.Is(err, storage.ErrDataNotFound)` ?

Reply:
Fixed